runtime/test-fw/src/ext_intr.S

```diff
@@ -0,0 +1,171 @@
+
+.section .init.text, "ax"
+.align 2
+_ext_intr_handler:
```

Comment:
I don't understand what we're testing here. What does this have to do with runtime firmware? It seems like this is more to do with testing the hardware model itself...

Reply:
Typically those tests go into hw-model/.

Reply:
Thanks that makes sense. I'll move it to hw-model. The goal is indeed to make the emulator more accurately match hardware.

Reply:
The runtime test firmware setup is more convenient than the test-harness rom as things like nmi is already set up. I can make the tests include another .S file that does all that ?